src/FileManagerIO/FileManager.java

```
@@ -0,0 +1,105 @@
+                    } else {
+                        System.out.println("Usage: cd <source> <target>");
+                    }
+                    break;
+                case "ls":
+                    listFiles();
+                    break;
+                case "pdw":
+                    printWorkingDirectory();
+                    break;
+                default:
+                    System.out.println("Unknown command " + firstCommand);
+                    break;
+            }
+        }
+    }
+
+    private void printWorkingDirectory() {
+        System.out.println(currentDirectory);
+    }
+
+    private void listFiles() {
+        File file = new File(currentDirectory);
+        File[] files = file.listFiles();
+        if (files != null) {
+            for (File f : files) {
+                System.out.println(f.getName());
+            }
+        }
+    }
+
+    private void changeDirectory(String path) {
+        String newPath;
+        if (path.startsWith("/")) {
+            newPath = path;
+        } else {
+            newPath = currentDirectory + "/" + path;
+        }
+        File file = new File(newPath);
+        if (file.isDirectory()) {
+            currentDirectory = newPath;
+        } else {
+            System.out.println("Error! " + newPath + " is not a directory");
+        }
+    }
+
+    private void copyFile(String source, String destination) {
+        File sourceFile = new File(source);
+        File destinatoinFile = new File(destination);
+
+        try (BufferedReader bufferedReader = new BufferedReader(new FileReader(sourceFile));
+             BufferedWriter bufferedWriter = new BufferedWriter(new FileWriter(destinatoinFile))) {
+            String line;
+            while ((line = bufferedReader.readLine()) != null) {
+                bufferedWriter.write(line);
+                System.out.println("File copied successfully.");
+            }
+        } catch (IOException e) {
+            e.printStackTrace();
```

throw an exception instead of printing a stack trace